packages/communication-react/review/communication-react.api.md

```diff
@@ -449,6 +454,11 @@ export type CallParticipantListParticipant = ParticipantListParticipant & {
     isSpeaking?: boolean;
 };
 
+// @beta
+export type CallParticipantLocator = {
```

Comment:
CallParticipantsLocator?

Reading the type name I assumed it can contain only 1 participant id.

Reply:
Good call, updated